src/pages/about/index.tsx

```diff
@@ -37,6 +35,33 @@ const About: NextPage = () => {
               development.
             </PageText>
 
+            <PriorityProjectsChart mb={6} color='brand.paragraph' />
```

Comment:
Not an issue, would suggest increasing the top and margin bottoms of the chart to add more space, maybe to `12`?

Reply:
```suggestion
            <PriorityProjectsChart my={12} color='brand.paragraph' />
```